config/initializers/amazon_s3.rb

```diff
@@ -0,0 +1,10 @@
+require 'aws-sdk-s3'
+
+# Aws.config.update({
+#                     region: 'us-west-2',
+#                     credentials: Aws::Credentials.new('akid', 'secret')
+#                   })
+
+creds = Minskrb::Application.credentials.aws
+Aws.config[:credentials] = Aws::Credentials.new(creds[:access_key_id], creds[:secret_access_key])
```

Comment:
You should configure your storage via `storage.yml` 
see https://edgeguides.rubyonrails.org/active_storage_overview.html#setup

Reply:
And then you can delete the file from initializers